src/app/check-pop-up/check-pop-up.component.html

```diff
@@ -0,0 +1,7 @@
+<h2 mat-dialog-title>{{data?.answer}}</h2>
```

Comment:
Based on the result of the answer, could you have the color of the pop up be green or red. You can pass styling specifications inside of the html so you can style based on conditions I think.